Analyse_Birds.qmd

```diff
@@ -1,1397 +1,1589 @@
+  )|>
+  dplyr::group_by(
+    Habitat
+    ) |>
+  dplyr::summarise(
+    count = n(),
+    prop_Nest_Fate = sum(Nest_Fate) / length(Nest_Fate),
+    mean_Init_Date = mean(Initiation_Date),
+    mean_Number_Fledged = mean(Number_Fledged),
+    mean_Nest_End_Date = mean(Nest_End_Date),
+  )
+
+## Calculation of the mean of each variables depending on the state
+data_state <- data |>
+  dplyr::select(
+    State,Nest_Fate,Initiation_Date, Number_Fledged, Nest_End_Date
+  )|>
+  dplyr::group_by(
+    State
+    ) |>
+  dplyr::summarise(
+    count = n(),
+    prop_Nest_Fate = sum(Nest_Fate) / length(Nest_Fate),
+    mean_Init_Date = mean(Initiation_Date),
+    mean_Number_Fledged = mean(Number_Fledged),
+    mean_Nest_End_Date = mean(Nest_End_Date),
+  )
+
+## Calculation of the mean of each variables depending on the county
+data_county <- data |>
+  dplyr::select(
+    State, County, Nest_Fate,Initiation_Date, Number_Fledged, Nest_End_Date
+  )|>
+  dplyr::group_by(
+    State, County
+    ) |>
+  dplyr::summarise(
+    count = n(),
+    prop_Nest_Fate = sum(Nest_Fate) / length(Nest_Fate),
+    mean_Init_Date = mean(Initiation_Date),
+    mean_Number_Fledged = mean(Number_Fledged),
+    mean_Nest_End_Date = mean(Nest_End_Date),
+  )
+data_sp
+```
+
+We create a list called data_summariser that will contains each previously created tables.
+
+```{r summarizedata}
+data_summariser <-list()  # Creation of the due list data_summariser
+data_summariser$Species <- data_sp # Add the Species table
+data_summariser$Habitat <- data_hab 
+data_summariser$State <- data_state
+data_summariser$County <- data_county
+
+data_summariser
+```
+
+Convert dates to months to make a MCA:
+
+```{r datesinmonth}
+### Creation of a function to replace the number by a month 
+
+num_to_month <- function(num){
+  if (num %in% (1:30)){
+    res <- "april"
+  } else if (num %in% (31:61)){
+    res <- "may"
+  } else if (num %in% (62:91)){
+    res <- "june"
+  } else if (num %in% (92:122)){
+    res <- "july"
+  } else if (num %in% (123:153)){
+    res <- "august"
+  } else {
+    res <- "september"
+  }
+  return (res)
+}
+```
+
+Now, we trasform all the quantitative variable into factors.
+
+```{r quantitativeintofactor}
+## Conversion into months 
+data_ACM <- data
+data_ACM$Initiation_Date <- lapply(X = data_ACM$Initiation_Date, 
+                                FUN = num_to_month
+                                )
+data_ACM$Nest_End_Date <- lapply(X = data_ACM$Nest_End_Date, 
+                              FUN = num_to_month
+                              )
+## Conversion into factors
+data_ACM <- data_ACM |>
+  dplyr::mutate(Nest_Fate = as.factor(Nest_Fate),
+                Initiation_Date = as.factor(unlist(Initiation_Date)),
+                Number_Fledged = as.factor(Number_Fledged),
+                Nest_End_Date = as.factor(unlist(Nest_End_Date)),
+                Year = as.factor(Year)
+                )
+```
+
+##### Manon part : MCA analysis (data exploring)
```

Comment:
You can remove "Manon part" after everyone have revieviewed the code,  I think it's not necessary to keep it , but thanks, it was easier to spot your part !